rmb-sdk-go/examples/client_direct/main.go

```diff
@@ -3,47 +3,91 @@ package main
 import (
 	"context"
 	"fmt"
+	"math/rand"
 	"os"
-	"time"
 
+	"github.com/google/uuid"
+	"github.com/rs/zerolog/log"
 	substrate "github.com/threefoldtech/tfchain/clients/tfchain-client-go"
+	"github.com/threefoldtech/tfgrid-sdk-go/rmb-sdk-go"
 	"github.com/threefoldtech/tfgrid-sdk-go/rmb-sdk-go/direct"
+	"github.com/threefoldtech/tfgrid-sdk-go/rmb-sdk-go/direct/types"
 )
 
+type resultsChan struct {
+	resutls chan bool
+}
+
 func app() error {
-	mnemonics := "<mnemonics goes here>"
+	mnemonics := "room illegal solar mosquito region mass alarm carpet stamp mutual hard inmate"
```

Comment:
mnemonics shouldn't be on the file, can be converted to os.Getenv or a command line argument.

Reply:
or keep as it was from before `"<mnemonic goes here>"`